app/controllers/categories_controller.rb

```diff
@@ -1,2 +1,24 @@
 class CategoriesController < ApplicationController
+
+	def index
+		@categories = Category.where("question_id = '#{params[:question_id]}'")
```

Comment:
No.

`Question.find(params[:question_id]).categories` is about 1_billion times easier to read.